docs/tokenomics/nns/proposal-requirements.md

```diff
@@ -0,0 +1,206 @@
+# Submitting a Proposal
+
+## Overview 
+
+### Submit a Proposal
+
+Only neurons with at least 10 ICP and 6 months of dissolve delay can submit proposals to the NNS. To learn more on how to submit a proposal, refer to the [How to Create an NNS motion proposal](https://wiki.internetcomputer.org/wiki/How-To:_Create_an_NNS_motion_proposal). 
+
+### Proposal Format
+
+Each proposal submitted to the NNS has the following fields:
+
+* **Summary:** text providing a short description of the proposal, composed using a maximum of 280 bytes.
+* **URL:** the web address of additional content required to evaluate the proposal, specified using HTTPS. For example, the address might describe content supporting the assignment of a DCID (data center id) to a new data center.
+* **Proposer:** the ID of the neuron that submitted the proposal. When a proposal is submitted, a “charge” is placed on its balance in case it is rejected. So the balance needs to be big enough to pay the charge on (all) rejection(s). The NNS requires a neuron to have a dissolve delay of ≥ 6 months to vote, and this applies to submitting proposals too.
```

Comment:
```suggestion
* **Proposer:** the ID of the neuron that submitted the proposal. When a proposal is submitted, a “charge” is placed on its balance in case it is rejected. The balance needs to be enough to pay the charge on (all) rejection(s). The NNS requires a neuron to have a dissolve delay of ≥ 6 months to vote, and which also applies to submitting proposals.
```